install.sh

```diff
@@ -11,15 +11,24 @@ mkdir -p "$BUILD_DIR"
 
 _pip_install() {
     if [[ -n "$CI" ]]; then
-        ccache -z
+        if [[ -n "${SCCACHE_BUCKET}" ]]; then
+            # Does sccache have flag for clear stats?
```

Comment:
It starts a daemon inside the container and the stats are per daemon run, which in this case is per build.